github-lab-bundle-spec.md

```diff
@@ -235,7 +235,7 @@ environment:
 
 attribute                        | required | type    | notes
 ---------------------------------| -------- | ------- | --------------------------------------
-startup_script.type              |          | string  | The type of startup script. Only `deployment_manager` is supported.
```

Comment:
What do you think of adding a link to go/ql-scripts? Up to you.

Reply:
I am a little hesitant because this is a public repo and the go link is only valid within Google. I think it may be better to someday move the content of go/ql-scripts to this repo.